app/src/main/java/com/example/replica/composables/WhatsAppScreen.kt

```diff
@@ -127,64 +201,71 @@ fun WhatsAppScreen(backHome: () -> Unit) {
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun NewMessage(value: String, onValueChange: (String) -> Unit) {
-    Spacer(modifier = Modifier.size(6.dp))
-    Row(
-        modifier = Modifier
-            .fillMaxWidth()
-            .padding(horizontal = 8.dp)
-    ) {
+    // In general, it is a good practice to only have 1 root-level Composable.
+    // Otherwise, the Composable that calls this one can influence how it looks!
+    // For example, if we called the original NewMessage inside a Row, the first Spacer would create
+    // horizontal space; if we called NewMessage inside a Column, the first Spacer would create
+    // vertical space! :S
+    Column(modifier = Modifier.wrapContentHeight()) {
```

Comment:
Nesting e.g. `Column`s is cheap in Compose, there's no performance problem with doing so.

Reply:
It might help to preview this file with whitespace changes ignored:

![CleanShot 2023-09-08 at 10 20 19@2x](https://github.com/FeKent/Replica/assets/6463980/90e9827e-37ed-48dc-b72c-99218c08a7d8)

The diff looks big, but it's because almost every line in the function was indented 4 spaces because of the new outer `Column`.